### DISCUSSION TITLE
启动项目问题

### DISCUSSION THREAD

**Discussion by hequndev** [fect-org/fect#133] [Q&A] [answered]
我尝试克隆该项目,查看package.json尝试运行构建该项目.但是出错了,是有什么注意事项吗?

**Comment by nonzzz** [comment, marked as answer]
@hequndev  我忘记更新contribute grid了 不好意思。启动项目的话先得在根目录下yarn 安装依赖。然后执行yarn clean 然后在执行yarn cli 在执行一次yarn 。可以参照`.travis.yml`的`script`部分。启动项目这一块我会找时间优化下。

**Reply by hequndev** [reply]
好的，非常感谢。

**Comment by takumi913** [comment]
@XeryYue Hey, man. 我根据 [contribute guide](https://github.com/fect-org/fect/blob/master/.github/CONTRIBUTING.zh-CN.md) 启动项目，终端里没有显示报错，但是网页没有显示，控制台报错如下:

![image](https://user-images.githubusercontent.com/38528079/146342689-4fed4a56-08e8-41e7-91bf-531a1e0a4f69.png)

这应该要怎么解决呢？

**Reply by nonzzz** [reply]
@zaunist  他的提示是缺少了vue-icon这个包。 你有在命令行运行`yarn icon`吗

**Reply by takumi913** [reply]
有的，我看了 package 目录，有 vue-icons 目录，也有 share2.js 文件
